src/types/resolveDescriptors.ts

```diff
@@ -136,7 +137,7 @@ export const toBounced = (type: string) => `${type}%%BOUNCED%%`;
 export function resolveTypeRef(ctx: CompilerContext, type: A.AstType): TypeRef {
     switch (type.kind) {
         case "type_id": {
-            const t = getType(ctx, idText(type));
+            const t = getType(ctx, idText(type), type.loc);
```

Comment:
```suggestion
            const t = getType(ctx, type);
```

and in similar places

Reply:
oh, indeed, fixed